google/cloud/storage/oauth2/service_account_credentials_test.cc

```diff
@@ -286,7 +286,7 @@ TEST_F(ServiceAccountCredentialsTest, ParseInvalidContentsFails) {
   std::string config = R"""( not-a-valid-json-string )""";
 
   auto actual = ParseServiceAccountCredentials(config, "test-as-a-source");
-  ASSERT_FALSE(actual.ok()) << "status=" << actual.status();
+  ASSERT_FALSE(actual) << "status=" << actual.status();
```

Comment:
FWIW, there's no need stream the status on a failed *false* assertion, because if the assertion fails, it's because the status was "ok", and streaming "OK" doesn't contain any useful information. So `ASSERT_FALSE(actual);` should be enough.

Reply:
Same point as above - I'd prefer fixing all occurrences of this in the g/c/s/oauth2 subdirectory in a separate PR.